sites/default/sqlconf.php

```diff
@@ -1,16 +1,17 @@
 <?php
 //  OpenEMR
 //  MySQL Config
-//  Referenced from /library/sqlconf.php.
-
-global $disable_utf8_flag;
-$disable_utf8_flag = false;
 
 $host	= 'localhost';
 $port	= '3306';
 $login	= 'openemr';
-$pass	= 'openemr';
-$dbase	= 'openemr';
+$pass	= '123456789';
```

Comment:
Rishabh,

Thanks for replying back.

You're going to want to undo your changes to the file. I would actually favor a "low tech" solution here by copying the following overtop your file and checking it in (thus, reverting your changes): https://raw.githubusercontent.com/openemr/openemr/master/sites/default/sqlconf.php

One tip I have for you is to put `sqlconf.php` on your Desktop and drag it into `sites/default/` when actively developing. Just be careful not to check it in.

Matthew